src/cloudwatch.ts

```diff
@@ -0,0 +1,148 @@
+import { ClientConfiguration as cwConfig, PutMetricDataInput } from "aws-sdk/clients/cloudwatch";
+import { ClientConfiguration as logConfig, FilterLogEventsRequest } from "aws-sdk/clients/cloudwatchlogs";
+import { CloudWatch, CloudWatchLogs } from "aws-sdk";
+import { DateTime } from "luxon";
+import { cwLogger } from "./handler";
+// @ts-ignore
+// tslint:disable-next-line:no-var-requires
+const AWSXRay = require("aws-xray-sdk");
```

Comment:
This may be an issue while you are doing local testing , since locally you would not have xray nor would you need it . The other services have a pattern to resolve this and the same can be done here.

Not a blocker, since you have to release today .. but would be a blocker otherwise, because it clearly proves local testing issues have not be addressed. Please address this next time

Reply:
Saw the same but this will only throw errors to the console, assuming you have the AWS_XRAY_CONTEXT_MISSING=LOG_ERROR environment variable set (as Jenkins does). Without the flag it will crash and burn.

Reply:
These particular lines will not be needed once type definitions are written for aws-xray-sdk